components/collection/SearchInput.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="relative w-full">
+    <input
+      v-if="!selectedCollection"
+      v-model="searchQuery"
+      type="text"
+      class="w-full h-12 p-2 border border-k-shade bg-background-color text-text-color focus:border-border-color outline-none"
+      :placeholder="$t('mint.collection.search')"
+      @input="handleSearch"
+    >
```

Comment:
can NeoAutocomplete be used here ?

Reply:
It's not that straightforward to convert. I will keep it as it is now.